.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,14 @@
 ## Requirements
+Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.
+Please note that we reserve the rights not to accept any domain at our own discretion.
```

Comment:
```suggestion

Please note that we reserve the rights not to accept any domain at our own discretion.
```

Reply:
@MaskDuck

Reply:
It makes it look better, so it's not a big blob of text no one will read. @MaskDuck